server/datastore/mysql/software_test.go

```diff
@@ -332,15 +337,26 @@ func TestLoadSupportsTonsOfCVEs(t *testing.T) {
 	require.NoError(t, ds.LoadHostSoftware(context.Background(), host))
 
 	sort.Slice(host.Software, func(i, j int) bool { return host.Software[i].Name < host.Software[j].Name })
-	require.NoError(t, ds.AddCPEForSoftware(context.Background(), host.Software[0], "somecpe"))
+
 	require.NoError(t, ds.AddCPEForSoftware(context.Background(), host.Software[1], "someothercpewithoutvulns"))
-	for i := 0; i < 1000; i++ {
-		part1 := rand.Intn(1000)
-		part2 := rand.Intn(1000)
-		part3 := rand.Intn(1000)
-		cve := fmt.Sprintf("cve-%d-%d-%d", part1, part2, part3)
-		require.NoError(t, ds.InsertCVEForCPE(context.Background(), cve, []string{"somecpe"}))
-	}
+
+	require.NoError(t, ds.withTx(context.Background(), func(tx sqlx.ExtContext) error {
```

Comment:
Here I just used a tx to optimize the batch-insertion a bit, reusing the cpe id instead of reading it on each call, and committing only once after all is done.